RELEASE_NOTES.md

```diff
@@ -27,6 +27,7 @@
 - For UPI Intent an error message will be shown when "Continue" button is pressed without selecting
   any UPI option.
 - For drop-in, improved accessibility of back/close button in the navigation bar.
+- For drop-in, show a toolbar on every intermediary screen, so shoppers can always easily navigate back.
```

Comment:
This file still has the old release notes

Reply:
Yes, I have to rebase the base branch. It will be fixed then